jobserver/views/users.py

```diff
@@ -48,13 +48,6 @@ def form_valid(self, form):
         user = User.objects.filter(email=form.cleaned_data["email"]).first()
 
         if user:
-            if not has_role(user, InteractiveReporter):
-                messages.error(
-                    self.request,
-                    "Only users who have signed up to OpenSAFELY Interactive can log in via email",
-                )
-                return redirect("login")
-
             if user.social_auth.exists():
```

Comment:
You might have spotted it by the time you see this comment, but the else on line 65 needs to become:

`elif InteractiveReporter not in user.all_roles:`